src/black/parsing.py

```diff
@@ -79,7 +91,7 @@ def lib2to3_parse(src_txt: str, target_versions: Iterable[TargetVersion] = ()) -
         src_txt += "\n"
 
     for grammar in get_grammars(set(target_versions)):
-        drv = driver.Driver(grammar, pytree.convert)
+        drv = driver.Driver(grammar)
```

Comment:
What is the motivation behind this change?

Reply:
Never mind it's explained in blib2to3 code.

Reply:
Ah yea the optimizations I made can be confusing! Profiling found that a lot of time parsing was spent in the Parser.push / Parser.shift methods so optimizing them was critical. Lookups at runtime are expensive and so are calls via the generic Python calling convention so I wanted to avoid them.